pyacq/devices/webcam_av.py

```diff
@@ -41,7 +41,7 @@ def run(self):
             for frame in packet.decode():
                 arr = frame.to_rgb().to_nd_array()
                 n += 1
-                self.out_stream.send(n, arr)
+                self.out_stream.send(arr, index=n)
```

Comment:
These devices don't have any way to skip indices, right? If that's the case, then there might be no need to provide the index at all (the stream will take care of it).